### DISCUSSION TITLE
Kibana 7.9.3

### DISCUSSION THREAD

**Discussion by paalaas** [fbaligand/kibana-enhanced-table#291] [Q&A] [answered]
I'm using Kibana 7.9.3 and tried version 1.13.3, where should I be able to set the csv encoding for the document table export?

**Comment by fbaligand** [comment]
Well, go to "Options" tab, and you will have a textbox for that:
![image](https://user-images.githubusercontent.com/3750300/192023939-e4c899a3-2d59-487a-a3dc-2ffd2fa75a2e.png)

**Reply by paalaas** [reply]
I see I don't see those options this is what I have available in 1.11.0 and 1.13.3 for Kibana 7.9.3
![image](https://user-images.githubusercontent.com/24648804/192036580-825faa92-98fe-4852-a5f8-d68317acc01b.png)

**Comment by fbaligand** [comment, marked as answer]
Very surprising…
Could you completely remove your browser cache (ctrl+shift+suppr) ? And then refresh your page?

**Reply by paalaas** [reply]
Actually ctrl+shift+del in Chrome

**Reply by fbaligand** [reply]
Sorry, I used the French word for « del » :)
